src/js/buttons/DefeatButton.js

```diff
@@ -0,0 +1,23 @@
+import { UITextButton, state } from "melonjs";
+import { params } from "../params";
+
+class DefeatButton extends UITextButton {
```

Comment:
If you want to refactor the victory and defeat buttons together, you can since it's mostly the same. It's up to you on how to resolve that and if you want to do it.

Also, if you want to test the functionality of the screens and it's the buttons only purpose, don't merge it to master. You can make the buttons in your own repository and test it. I would remove the buttons from this pull request.